src/proxy.c

```diff
@@ -54,6 +54,12 @@ int proxy_process(ProxyRequest *request, ProxyReply *reply)
         case P_GET_BOOTARGS:
             reply->retval = boot_args_addr;
             break;
+        case P_GET_BOOT_CPU_IDX:
+            reply->retval = boot_cpu_idx;
+            break;
+        case P_GET_BOOT_CPU_MPIDR:
+            reply->retval = boot_cpu_mpidr;
```

Comment:
I don't think anyone has plans to use the non boot CPU for proxy commands so this is equivalent to `u.mrs(MPIDR_EL1)`

Reply:
Yes, i'l remove